src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -528,12 +531,16 @@ public async Task<IActionResult> VerifyOrganisationDetails()
 
             if (ProviderEligibleToChangeRoute(roatpRegisterStatus))
             {
+                if (applicationDetails?.RoatpRegisterStatus?.ProviderTypeId != null)
+                {
+                    return await TermsAndConditions(new SelectApplicationRouteViewModel { ApplicationRouteId = applicationDetails.RoatpRegisterStatus.ProviderTypeId.Value });
```

Comment:
Shouldn't this be a RedirectToAction so the URL is correct?  It's showing `..\choose-provider-route` instead of `..\terms-conditions-making-application'

Reply:
good spot, fixed